packages/web/src/index.css

```diff
@@ -4,6 +4,23 @@
 @tailwind components;
 @tailwind utilities;
 
+@layer base {
```

Comment:
I think there is a section lower in the file with other `.cm-X` stuff, can we move that there?

Reply:
@nichochar moved down with others